dataland-frontend/tests/e2e/fixtures/custom_mocks/AllEuNonFinancialsDataServerResponse.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  type DataAndMetaInformationEuTaxonomyDataForNonFinancials,
+  type EuTaxonomyDataForNonFinancials,
+  type EuTaxonomyDetailsPerCashFlowType,
+} from "@clients/backend";
+import { DataMetaInformationGenerator } from "@e2e/fixtures/data_meta_information/DataMetaInformationFixtures";
+import { EuNonFinancialsGenerator } from "@e2e/fixtures/eutaxonomy/non-financials/EuTaxonomyDataForNonFinancialsFixtures";
+import { generateCurrencyValue, generatePercentageValue } from "@e2e/fixtures/common/NumberFixtures";
+import { DEFAULT_PROBABILITY } from "@e2e/utils/FakeFixtureUtils";
+import { generateNaceCodes } from "@e2e/fixtures/common/NaceCodeFixtures";
+import { generateDataPoint, generateReferencedReports } from "@e2e/fixtures/common/DataPointFixtures";
+import { generateEuTaxonomyWithBaseFields } from "@e2e/fixtures/eutaxonomy/EuTaxonomySharedValuesFixtures";
+import { generateCurrencyCode } from "@e2e/fixtures/common/CurrencyFixtures";
+import { generateArray } from "@e2e/fixtures/FixtureUtils";
+import { range } from "@/utils/ArrayUtils";
+
+class MinimumAcceptedEuNonFinancialsGenerator extends EuNonFinancialsGenerator {
```

Comment:
Docstring could give a bit more detail what "MinimumAccepted" means in the context of this class

Reply:
I added a docstring.